Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -606,6 +607,16 @@ public final class PackageBuilder {
                     return potentialModuleMap[name]
                 }
             })
+            // If there are plugin usages, consider them to be dependencies too.
+            if let pluginUsages = target.pluginUsages {
+                successors += pluginUsages.compactMap({
+                    switch $0 {
+                    case .plugin(let name, let package):
+                        return (package == nil) ? potentialModuleMap[name] : nil
```

Comment:
should this be case-sensitive or case-insensitive?

Reply:
That's a good question — I did the same thing as what's being done earlier for the regular dependencies, so if we make it case-insensitive here, we should change it for regular dependencies too.  So that would be a semantic change that I don't think we should make in this PR.

Reply:
IMO we should consider such change, but I agree a separate PR makes sense

Reply:
Right, making target dependencies case-insensitive seems like a different consideration that should probably be tied to tools version.  Regardless, I think that dependencies and plugin usages should be consistent.  We can file a tracking Radar on the case sensitivity.